pkg/consensus/recovery_message.go

```diff
@@ -0,0 +1,160 @@
+package consensus
+
+import (
+	"github.com/CityOfZion/neo-go/pkg/io"
+	"github.com/CityOfZion/neo-go/pkg/util"
+	"github.com/pkg/errors"
+)
+
+type (
+	// recoveryMessage represents dBFT Recovery message.
+	recoveryMessage struct {
+		PreparationHash     *util.Uint256
+		PreparationPayloads map[uint16]*preparationCompact
+		CommitPayloads      map[uint16]*commitCompact
+		ChangeViewPayloads  map[uint16]*changeViewCompact
```

Comment:
Do you really need maps for these?

Reply:
Actually no, maps were in C# implementation. Slices look better.